tests/acceptance/config/behat.yml

```diff
@@ -46,6 +46,7 @@ default:
         - SearchContext:
         - TrashbinContext:
         - WebDavPropertiesContext:
+        - WebDavLockingContext:
```

Comment:
https://drone.owncloud.com/owncloud/ocis/15181/40/7
`WebDavLockingContext` context is not found in the suite environment. Have you registered it?

A few other suites also need to know about `WebDavLockingContext`